The way you're exporting messes up typescript type resolution for callers

**Describe the bug**
Here in your index.ts file:
https://github.com/uport-project/uport-credentials/blob/develop/src/index.ts#L8

You are mapping your classes to const's then exporting those. This essentially messes up TypeScripts type inference and treats them as values rather than types.

**To Reproduce**
Steps to reproduce the behavior:
1. `npm i uport-credentials`
2. create index.ts and add the following contents:
```ts
import { Credentials } from 'uport-credentials'
class Example {
  private credentials: Credentials
  constructor() {
    this.credentials = new Credentials({
      did: 'did:ethr:0x31486054a6ad2c0b685cd89ce0ba018e210d504e',
      privateKey: 'ef6a01d0d98ba08bd23ee8b0c650076c65d629560940de9935d0f46f00679e01'
    })
}
```
3. Either use `tsc` to compile the code or open in VSCode to see code errors
4. See error:
```
src/auth/uport.service.ts:8:24 - error TS2749: 'Credentials' refers to a value, but is being used as a type here.

8   private credentials: Credentials
                         ~~~~~~~~~~~
```

**Expected behavior**
I expect Credentials to be exported as a class and therefore useable as a type.

**Screenshots**
![2019-06-17](https://user-images.githubusercontent.com/10974/59650718-a0a1a180-914c-11e9-8b12-3a4c600febe9.png)

**Desktop (please complete the following information):**
 - OS: Windows
 - Nodejs, tsc

**Additional context**
I believe the problem is in index.ts you are referencing your exported types as constants which typescript then interprets as values and then exporting those. Instead I would just export the classes directly.

So:
```ts
import Creds from './Credentials'
import { SimpleSigner as S } from 'did-jwt'
import { ContractFactory as CF } from './Contract'

export const Credentials = Creds
export const SimpleSigner = S
export const ContractFactory = CF
export default { Credentials, SimpleSigner, ContractFactory }
```
Should be:
```ts
export Credentials from './Credentials'
export { SimpleSigner } from 'did-jwt'
export { ContractFactory } from './Contract'
```

--- a/src/index.ts
+++ b/src/index.ts
@@ -1,8 +1,5 @@
-import Creds from './Credentials'
-import { SimpleSigner as S } from 'did-jwt'
-import { ContractFactory as CF } from './Contract'
+import Credentials from './Credentials'
+import { SimpleSigner } from 'did-jwt'
+import { ContractFactory } from './Contract'
 
-export const Credentials = Creds
-export const SimpleSigner = S
-export const ContractFactory = CF
-export default { Credentials, SimpleSigner, ContractFactory }
+export { Credentials, SimpleSigner, ContractFactory }

